ple/games/donkeykong/__init__.py

```diff
@@ -0,0 +1,218 @@
+
+        # Create fireballs as required, depending on the number of Donkey Kongs in our game at the moment
+        if self.fireballTimer == 0:
+            self.newGame.CreateFireball(self.newGame.Enemies[0].getPosition(), 0)
+        elif len(self.newGame.Enemies) >= 2 and self.fireballTimer == 23:
+            self.newGame.CreateFireball(self.newGame.Enemies[1].getPosition(), 1)
+        elif len(self.newGame.Enemies) >= 3 and self.fireballTimer == 46:
+            self.newGame.CreateFireball(self.newGame.Enemies[2].getPosition(), 2)
+        self.fireballTimer = (self.fireballTimer + 1) % 70
+
+        # Animate the coin
+        for coin in self.coinGroup:
+            coin.animateCoin()
+
+        # To check collisions below, we move the player downwards then check and move him back to his original location
+        self.newGame.Players[0].updateY(2)
+        self.laddersCollidedBelow = self.newGame.Players[0].checkCollision(self.ladderGroup)
+        self.wallsCollidedBelow = self.newGame.Players[0].checkCollision(self.wallGroup)
+        self.newGame.Players[0].updateY(-2)
+
+        # To check for collisions above, we move the player up then check and then move him back down
+        self.newGame.Players[0].updateY(-2)
+        self.wallsCollidedAbove = self.newGame.Players[0].checkCollision(self.wallGroup)
+        self.newGame.Players[0].updateY(2)
+
+        # Sets the onLadder state of the player
+        self.newGame.ladderCheck(self.laddersCollidedBelow, self.wallsCollidedBelow, self.wallsCollidedAbove)
+
+        for event in pygame.event.get():
+            # Exit to desktop
+            if event.type == QUIT:
+                pygame.quit()
+                sys.exit()
+
+            if event.type == KEYDOWN:
+                # Get the ladders collided with the player
+                self.laddersCollidedExact = self.newGame.Players[0].checkCollision(self.ladderGroup)
+
+                if (event.key == self.actions["jump"] and self.newGame.Players[0].onLadder == 0) or (
+                        event.key == self.actions["up"] and self.laddersCollidedExact):
+                    # Set the player to move up
+                    self.direction = 2
+                    if self.newGame.Players[0].isJumping == 0 and self.wallsCollidedBelow:
+                        # We can make the player jump and set his currentJumpSpeed
+                        self.newGame.Players[0].isJumping = 1
+                        self.newGame.Players[0].currentJumpSpeed = 7
+
+        # Update the player's position and process his jump if he is jumping
+        self.newGame.Players[0].continuousUpdate(self.wallGroup, self.ladderGroup)
+
+        '''
+        We use cycles to animate the character, when we change direction we also reset the cycles
+        We also change the direction according to the key pressed
+        '''
+        keyState = pygame.key.get_pressed()
+        if keyState[self.actions["right"]]:
```

Comment:
@ntasfi  Doing this does not work for me, if I put it in the KEYDOWN type, then the event gets fired only once per keypress, so the player does not move when the key is continuously being pressed. (I will update the PR anyways)

I have removed the score and life displays.